10up/ElasticPress
# Add Comments block.

### Description of the Change
Adds a Block version of the Comments widget as the "Search Comments" block.

The block replicates the functionality of the Comments widget with the addition of an editable `<label>` element for the search form.


Closes #2972

### How to test the Change
1. Open the blocks-based widgets editor.
2. Click to insert a block, search for the Search Comments (ElasticPress) block.
3. The block should appear and function the same as the legacy Widget.
4. With the Classic Widgets plugin enabled, add the Comments &mdash; ElasticPress widget.
5. Disable Classic Widgets and open the blocks-based widgets editor.
6. It should be possible to transform the legacy widget into the new block.

### Changelog Entry
- Added: A Search Comments block for searching comments. Replaces the Comments widget in the block editor.

### Credits
@JakePT 


### Checklist:
